libs/agora/services/jest.config.ts

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  displayName: 'web-services',
```

Comment:
Set to the name of the project, i.e. agora-services?

Reply:
good catch!  I will update this.